packages/schema/lib/schemas/FieldSchema.js

```diff
@@ -120,6 +120,10 @@ module.exports = makeSchema(
           'Does the value of this field affect the definitions of other fields in the set?',
         type: 'boolean',
       },
+      steadyState: {
+        description: 'Prevents triggering on new output until all values for fields with this property remain unchanged for 2 polls. Can be used to, e.g., not trigger on a new contact until the contact has completed typing their name.',
```

Comment:
Do you think it would be worthwhile to include a note about how this only applies to "Polling Triggers"?

Reply:
Yes, it would be helpful. Will include it.

Reply:
Added